Resources/Prototypes/DeltaV/Entities/Objects/Weapons/Guns/Revolvers/revolvers.yml

```diff
@@ -0,0 +1,67 @@
+- type: entity
+  name: fitz special
+  parent: [BaseWeaponRevolver, BaseC1Contraband]
+  id: WeaponRevolverFitz
+  description: A compact and concealable self defence snub revolver. Uses .38 special ammo.
```

Comment:
Misleading description: It states the gun uses .38 special, which is not present in the Frontier codebase. In reality, it uses .45 magnum.